driver-core/src/main/java/com/datastax/driver/core/TableOptionsMetadata.java

```diff
@@ -183,11 +182,12 @@
       this.extensions = ImmutableMap.copyOf(row.getMap(EXTENSIONS, String.class, ByteBuffer.class));
     else this.extensions = ImmutableMap.of();
 
-    if (is380OrHigher) this.cdc = isNullOrAbsent(row, CDC) ? DEFAULT_CDC : row.getBool(CDC);
-    else this.cdc = DEFAULT_CDC;
+    this.cdc = isNullOrAbsent(row, CDC) ? DEFAULT_CDC : row.getBool(CDC);
 
-    if (is400OrHigher) this.additionalWritePolicy = row.getString(ADDITIONAL_WRITE_POLICY);
-    else this.additionalWritePolicy = DEFAULT_ADDITIONAL_WRITE_POLICY;
+    this.additionalWritePolicy =
+        isNullOrAbsent(row, ADDITIONAL_WRITE_POLICY)
+            ? DEFAULT_ADDITIONAL_WRITE_POLICY
+            : row.getString(ADDITIONAL_WRITE_POLICY);
```

Comment:
One (quite possibly minor) point: we're losing a bit of insurance against aberrant cases by excluding the version guards, right?  The original code _expected_ fields to be present for certain versions.  The new code will silently handle missing fields silently due to isNullOrAbsent() even if a field should be present for a specific version.

This should only matter if, say, a server bug causes expected fields to not be returned for some reason.  I'm merely mentioning it to say we do lose that notion of version-based expectation here.

Reply:
On a related point: it seems a bit odd that after this change some fields still have version guards while some don't.  Perhaps we should make this change wholesale to all fields examined by this class?

Reply:
> I'm merely mentioning it to say we do lose that notion of version-based expectation here.

Correct, but as you pointed out, I don't see how this would matter unless the server is misbehaving.

> it seems a bit odd that after this change some fields still have version guards while some don't.

Indeed, but for these fields, we distinguish 3 states: 

1. the option is supported and present: use the value in the row
2. the option is supported but not present: use the default value instead
3. the option is not supported: use null

The version guard is required for item 3.

Reply:
Fair point on the remaining version bounds.  And I don't feel super-strongly about the fact that we're no longer considering whether a field _should be_ provided for the changes here.  I agree that it should only figure in when the server is misbehaving... my concern was that if that were to happen it might be better for my client to fail loudly.

Regardless, I should've prefixed all of this with the "nit" label; none of it is fatal.  What you have should be fine.